Procurement/Controls/ItemDisplay.xaml.cs

```diff
@@ -183,6 +186,8 @@ private void BindSocketPopup(ItemDisplayViewModel vm)
             };
         }
 
+        private SetBuyoutView buyoutControl;
```

Comment:
Why does this need to be a field?

Reply:
getContextMenu will assign it during ItemDisplay_MouseRightButtonUp.

Once the popup has been generated, I use this private field to cast the DataContext to get the ViewModel to assign IsDataInClipboard back to false so that the subsequent displays of the popup which have had the data copied to the clipboard will show the button for subsequent exports.

See:

https://github.com/Stickymaddness/Procurement/blob/1b950ec148e9e913e9fa0716c645d20d2ff4f1d3/Procurement/Controls/ItemDisplay.xaml.cs#L56